src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/IndentBlockFormattingRule.cs

```diff
@@ -257,7 +257,14 @@ private static void AddBracketIndentationOperation(List<IndentBlockOperation> li
         if (!bracketPair.IsValidBracketOrBracePair())
             return;
 
-        if (node.Parent != null && node.Kind() is SyntaxKind.ListPattern or SyntaxKind.CollectionExpression)
+        if (node.Parent != null && node.Kind() is SyntaxKind.ListPattern)
+        {
+            AddAlignmentBlockOperationRelativeToFirstTokenOnBaseTokenLine(list, bracketPair);
```

Comment:
💭 What happens if we omit this line?

What happens if we remove the `SetAlignmentBlockOperation` call for `CollectionExpression`?

What happens if we change the call for `CollectionExpression` to use the same form we have for `ListPattern`?